include/zmq.h

```diff
@@ -638,6 +631,18 @@ ZMQ_EXPORT int   zmq_timers_reset (void *timers, int timer_id);
 ZMQ_EXPORT long  zmq_timers_timeout (void *timers);
 ZMQ_EXPORT int   zmq_timers_execute (void *timers);
 
+/******************************************************************************/
+/*  GSSAPI socket options to set name type                                    */
+/******************************************************************************/
+
+#define ZMQ_GSSAPI_PRINCIPAL_NAMETYPE 1090
```

Comment:
I think it's better to have the definitive value immediately (90, 91) so that when it's declared stable folks who are already using it with DRAFT flag will not have any breakage, it will just keep working - a bit less disruption